src/styles/App.scss

```diff
@@ -1 +1,6 @@
-@import "custom";
\ No newline at end of file
+@import "custom";
+
+html, body, main, #root, #app, #app > div {
```

Comment:
Add this either to the custom css file or to the globalStyles file - we don't want to create another layer of styles in another file we have to trace through